.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -78,35 +78,43 @@ commands:
       - run:
           name: Monitoring Setup
           command: |
+              # config params
+              toolPath="<< parameters.tool_path >>"
+              actionNS="<< parameters.action_namespace >>"
+              actionPackage="<< parameters.action_package >>"
+              actionName="<< parameters.action_name >>"
+              nrAuth="<< parameters.newrelic_auth >>"
+              nrName="<< parameters.newrelic_name >>"
+              nrURL="<< parameters.newrelic_url >>"
+              nrGroupPolicy="<< parameters.newrelic_group_policy >>" 
+              spAuth="<< parameters.statuspage_auth >>"
+              spPageID="<< parameters.statuspage_page_id >>"
+              spName="<< parameters.statuspage_name >>"
+              spGroup="<< parameters.statuspage_group >>"
+
               # statuspage automation
-              spCmd="node << parameters.tool_path >>/statuspage setup --silent"
-              if [ "<< parameters.statuspage_name >>" != "" ]; then
-                spCmd="${spCmd} --name \"<< parameters.statuspage_name >>\""
+              spCmd="node ${toolPath}/statuspage setup --silent"
+              if [ "${spName}" != "" ]; then
+                spCmd="${spCmd} --name \"${spName}\""
               fi
-              if [ "<< parameters.statuspage_group >>" != "" ]; then
-                spCmd="${spCmd} --group \"<< parameters.statuspage_group >>\""
+              if [ "${spGroup}" != "" ]; then
+                spCmd="${spCmd} --group \"${spGroup}\""
               fi
-              email="`eval ${spCmd}`"
+              spEmail="`eval ${spCmd}`"
```

Comment:
If this is `bash` you could use:
```
eval ${spCmd} ${spName:+--name \"${spName}\"} ${spGroup:+--group \"${spGroup}\"}
```
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Nice, thanks! Yes, it's `bash` 😅